### ISSUE TITLE
Multipass feature

### ISSUE THREAD

**Issue by shemchavez** [Shopify/mobile-buy-sdk-ios#946]
Hi Shopify Team, I've browsed this library and haven't found **MultiPass** implementation? Is it already available? Or I have missed it? Thanks

**Comment by morganmccunn** [comment]
Hey @shemchavez - not yet! But we're investigating this so I'll circle back as soon as it's available.

**Comment by shemchavez** [comment]
Thanks @morganmccunn ! 👍

**Comment by aytwit** [comment]
Hey @morganmccunn - is there any update on this?

**Comment by cocoahero** [comment]
Yeah, not having this functionality is a game changer for us currently.

**Comment by naeemshaikh90** [comment]
Hey @morganmccunn and @cocoahero, Recently I have implemented [Multipass](https://help.shopify.com/en/api/reference/plus/multipass) implementation in Swift using [CryptoSwift](https://github.com/krzyzanowskim/CryptoSwift) library. Can you guys help me to integrate into SDK? I have read [this](https://github.com/Shopify/mobile-buy-sdk-ios/blob/master/.github/CONTRIBUTING.md), but not sure how and where will it fit exactly.

**Comment by DrTeePot** [comment]
Hello all,

- Multipass can be used with web checkouts by following [this guide](https://shopify.dev/tutorials/create-a-checkout-with-storefront-api#associating-the-customer).
- Multipass can be used to get a customer access token in the `2020-04` version of the API. See [this changelog post](https://shopify.dev/changelog/new-customeraccesstokencreatewithmultipass-mutation). This SDK will be updated shortly after the API is released.
